script.js

```diff
@@ -0,0 +1,69 @@
+let developers = [
+    {
+        "name": "Carolin Bloeme",
+        "id": "carolin-bloeme"
+    },
+    {
+        "name": "Sun Jun",
+        "id": "sun-jun"
+    },
+    {
+        "name": "Song Bao",
+        "id": "song-bao"
+    },
+    {
+        "name": "Olivia Arribas",
+        "id": "olivia-arribas"
+    },
+    {
+        "name": "Samah abu laymun ",
+        "id": "samah-abu-laymun"
+    }
+]
+
+let ul = document.getElementById('developers');
```

Comment:
I think there is no need for this one as you add the content directly (Duplicated)